design/META.json

```diff
@@ -2169,6 +2169,42 @@
             "tags": ["pressure", "gauge", "psi", "pipe"],
             "author": "Eric Atha (PX Blue)",
             "size": 24
+        },
+        {
+            "name": "Battery Reset",
+            "filename": "battery_reset.svg",
+            "family": ["Devices"],
+            "style": "baseline",
+            "tags": ["battery", "test", "reset"],
+            "author": "Huayun Huang (PX Blue)",
+            "size": 24
+        },
+        {
+            "name": "Battery Reset Outline",
+            "filename": "battery_reset_outline.svg",
+            "family": ["Devices"],
+            "style": "baseline",
+            "tags": ["battery", "test", "reset", "outline"],
+            "author": "Amy Deng",
+            "size": 24
+        },
+        {
+            "name": "Battery Large",
+            "filename": "battery_large.svg",
+            "family": ["Devices"],
+            "style": "baseline",
+            "tags": ["battery"],
+            "author": "Amy Deng",
+            "size": 24
+        },
+        {
+            "name": "Technician",
+            "filename": "technician.svg",
+            "family": ["General"],
+            "style": "baseline",
+            "tags": ["technician", "electrician", "service", "person", "account hard hat"],
+            "author": "Austin Andrews",
+            "size": 24
```

Comment:
Do we usually include a link to these people or a link to the original icon location on material design icons? Like "Austing Andrews (hyperlink here)"?

Reply:
Historically no. Do you want me to log an issue there?

Reply:
https://github.com/pxblue/icons/blob/dev/design/META.json#L73

Reply:
and for custom ones that we do it's usually "First Last (PX Blue)"

Reply:
Ah, I see. I thought you meant a deeplink into the icon. Will update.

Amy is not part of PX Blue. Do you still want to do "PX Blue" or "DPQ ID" or "DPQ" or...?

Reply:
A couple things... for tags on the batteries I'd add "lead acid", "batteries" to help with searching. I'd also do a version without the +,- (both filled and outlined) so that in the future we can add variations that match our typical battery icon alternates (charging, levels, etc.)

Reply:
I originally was thinking a deep link, but I think you're right that we haven't done that anywhere else.

I would still say PX Blue for Amy. Count it as an innersource :)

Reply:
It also looks like there is a difference in corner radius on the terminals (the filled reset icon is rounder)

Reply:
As for naming... "Large" might be okay, but it's not necessarily true. Most modern (li) batteries are made up of a series of cells that look like our "AA" style battery. It's an abstraction, but also accurate.

Reply:
eh..the _icon_ is large, not necessarily the battery